api/v1alpha1/condition_consts.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
This should be VMware and consistent with the other files

```
// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I guess changing this file is probably fine, but not for condition_types.go since it is basically straight from cluster-api